roles/bwc/tasks/ldap.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Setup st2.conf auth backend to LDAP
+  become: yes
+  ini_file:
+    dest: "{{ st2_config_file_path }}"
+    section: auth
+    option: backend
+    value: ldap
+    backup: yes
+
+- name: Setup st2.conf auth backend_kwargs for LDAP
+  become: yes
+  ini_file:
+    dest: "{{ st2_config_file_path }}"
+    section: auth
+    option: backend_kwargs
+    value: "{{ ldap.config | to_nice_json }}"
```

Comment:
This prints out a multi-line string in the ini file and our config parser pukes. I tried to_json. Still the same.